### PR TITLE
[SPARK-44290][CONNECT][FOLLOW-UP] Skip flaky tests, and fix a typo in session UUID together

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR is a followup of https://github.com/apache/spark/pull/41495 that skips a couple of flaky tests. In addition, this PR fixes a typo together.

### Why are the changes needed?

To keep the tests green. In order to reenable the tests, it needs other fixes together that might refactor the whole test cases which takes a while. I will followup and fix them in SPARK-44348

### Does this PR introduce _any_ user-facing change?

No, the feature is not released to end users yet.

### How was this patch tested?

Unittests skipped for now.

### CODE REVIEW

**Review by zhengruifeng — APPROVED** [review]

**Review by LuciferYang — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

File: python/pyspark/sql/tests/connect/client/test_artifact.py [hunk 1]
```diff
@@ -245,6 +245,7 @@ def func(x):
             spark_session.addArtifacts(pyfile_path, pyfile=True)
             self.assertEqual(spark_session.range(1).select(func("id")).first()[0], 10)
 
+    @unittest.skip("SPARK-44348: Reeanble Session-based artifact test cases")
```
Comment (yaooqinn) [inline python/pyspark/sql/tests/connect/client/test_artifact.py:248]:
```suggestion
    @unittest.skip("SPARK-44348: Reenable Session-based artifact test cases")
```

File: python/pyspark/sql/tests/connect/client/test_artifact.py [hunk 2]
```diff
@@ -272,6 +273,7 @@ def func(x):
             spark_session.addArtifacts(f"{package_path}.zip", pyfile=True)
             self.assertEqual(spark_session.range(1).select(func("id")).first()[0], 5)
 
+    @unittest.skip("SPARK-44348: Reeanble Session-based artifact test cases")
```
Comment (yaooqinn) [inline python/pyspark/sql/tests/connect/client/test_artifact.py:276]:
```suggestion
    @unittest.skip("SPARK-44348: Reenable Session-based artifact test cases")
```

File: python/pyspark/sql/tests/connect/client/test_artifact.py [hunk 3]
```diff
@@ -303,6 +305,7 @@ def func(x):
             spark_session.addArtifacts(f"{archive_path}.zip#my_files", archive=True)
             self.assertEqual(spark_session.range(1).select(func("id")).first()[0], "hello world!")
 
+    @unittest.skip("SPARK-44348: Reeanble Session-based artifact test cases")
```
Comment (yaooqinn) [inline python/pyspark/sql/tests/connect/client/test_artifact.py:308]:
```suggestion
    @unittest.skip("SPARK-44348: Reenable Session-based artifact test cases")
```

File: python/pyspark/sql/tests/connect/client/test_artifact.py [hunk 4]
```diff
@@ -328,6 +331,7 @@ def func(x):
             spark_session.addArtifacts(file_path, file=True)
             self.assertEqual(spark_session.range(1).select(func("id")).first()[0], "Hello world!!")
 
+    @unittest.skip("SPARK-44348: Reeanble Session-based artifact test cases")
```
Comment (yaooqinn) [inline python/pyspark/sql/tests/connect/client/test_artifact.py:334]:
```suggestion
    @unittest.skip("SPARK-44348: Reenable Session-based artifact test cases")
```

**Comment by HyukjinKwon** [comment]
Thank you @yaooqinn . I am merging it in - the tests passed, and the last changes are just typos in the message.

Merged to master.

**Pull request closed without merging**